test/controllers/util/DisplayPropertiesUtilsSpec.scala

```diff
@@ -17,17 +17,17 @@ class DisplayPropertiesUtilsSpec extends AnyFlatSpec with MockitoSugar with Befo
   private val displayPropertiesUtils = new DisplayPropertiesUtils(displayProperties, customMetadata)
 
   "convertPropertiesToFormFields" should "return the list of properties requested in ordinal order" in {
-    val fields: Seq[FormField] = displayPropertiesUtils.convertPropertiesToFormFields
+    val fields: Seq[FormField] = displayPropertiesUtils.convertPropertiesToFormFields(displayProperties)
 
-    fields.size should equal(3)
-    fields.head.fieldId should equal("Dropdown1")
-    fields.tail.head.fieldId should equal("Dropdown2")
-    fields.last.fieldId should equal("Dropdown3")
+    fields.size should equal(6)
```

Comment:
You can use the `.find` function on the list to find all the individual fields and check their properties, eg:
```
val closurePeriodField: FormField = fields.find(_.fieldName == "ClosurePeriod").get
closurePeriodField.fieldId should equal ... etc ...
```

Reply:
Since the test is checking if the properties are returned in ordinal order I've gone with something like `fields should equal(List("FoiExemptionAsserted", "ClosureStartDate", "ClosurePeriod", "Radio", "TestProperty2", "Dropdown"))`